Translations/ru.po

```diff
@@ -924,6 +925,7 @@ msgstr "Умение"
 msgid "{:s} Skill"
 msgstr "{:s} навык"
 
+# Why "чары", but "заклинание"?
 #: Source/control.cpp:771
 msgid "Spell"
 msgstr "Чары"
```

Comment:
because it's the one displayed on spell
![image](https://user-images.githubusercontent.com/14297035/140610408-c0d5537f-4372-49e0-8d7f-1c865fe111c7.png)

"spella" on this picture - so it has to be super short to fit

Reply:
@qndel I'm not sure if using different (although synonymic) terms in various parts of the UI is the best solution. Can we use the abbreviations?

Reply:
🤷

Reply:
> @qndel I'm not sure if using different (although synonymic) terms in various parts of the UI is the best solution. Can we use the abbreviations?

I am not certain we can abbreviate everything though and keep understanding. заклинание for example might be too long. Чары I think is fine though.

Reply:
@Iniquitatis Speedbook only fits 5 characters. I tried "Заклин" shortening previously, but @AAS didn't like it and proposed "Чары"